packages/babel-plugin-proposal-partial-application/test/fixtures/general/member-expression/output.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+var _param, _o, _o$f, _param2, _param3, _p, _p$b, _a$b$c$d$e, _a$b$c$d$e$foo;
+
+const g = (_o = o, _o$f = o.f, _param = x, function f(_argPlaceholder) {
+  return _o$f.call(_o, _argPlaceholder, _param, 1);
+});
+const h = (_p = p, _p$b = p.b, _param2 = y, _param3 = x, function b(_argPlaceholder2) {
+  return _p$b.call(_p, 1, _param2, _param3, 2, _argPlaceholder2);
+});
+const j = (_a$b$c$d$e = a.b.c.d.e, _a$b$c$d$e$foo = a.b.c.d.e.foo, function foo(_argPlaceholder3) {
```

Comment:
This should be

```js
const j = (_a$b$c$d$e = a.b.c.d.e, _a$b$c$d$e$foo = _a$b$c$d$e.foo, ...
```

because `a.b.c.d.e` could be getters which must be accessed exactly once.